CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 2021.01.1
+
+**Released**: 2020.01.26
+
+**Commit Delta**: [Change from 2021.01.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
```

Comment:
@oblandford-novetta sorry this was my bad, the "change from" text was correct before. i meant to comment on the "manifests" link.

```suggestion
**Commit Delta**: [Change from 2020.12.1 release](https://github.com/plus3it/spel/compare/2020.12.1...2021.01.1)
```

Reply:
OK. That is my bad as well. I should have been paying more attention when making the suggested update. I was just like a robot. I also should have done all of this with a find replace in the first place.